app/src/main/java/com/android/unio/ui/map/Map.kt

```diff
@@ -121,16 +122,20 @@ fun MapScreen(
         requestPermissions()
       }
     }
-    cameraPositionState.position =
-        CameraPosition.fromLatLngZoom(userLocation ?: EPFL_COORDINATES, INITIAL_ZOOM_LEVEL)
   }
 
-  // Center map on the user's location initially if available
+  // Center map on the center location initially if available
   LaunchedEffect(userLocation) {
-    if (userLocation != null && !initialCentered) {
```

Comment:
I don't think removing this boolean is a good idea, as if the `userLocation` state flow is updated, e.g. the user moves to a different location and their location is updated in the view model, the map would automatically recenter on one of the logics below (either centers on the user's location, event's location or on EPFL's location) for no reason, and that's a behavior we don't want.